server/dba/migrations/49-addDistricts.js

```diff
@@ -0,0 +1,29 @@
+'use strict';
+
+let doTransaction = require('./modules/transaction');
+
+const TABLE_NAME = 'fsForests';
+
+module.exports = {
+  up: function(queryInterface, Sequelize) {
+    let operations = [
+      {
+        operation: 'add',
+        field: 'districts',
+        type: Sequelize.JSONB
```

Comment:
I'm not super familiar with this `Sequalize.JSONB` data type, but I pulled it down and ran some migrations, and everything looked g2g.